test/tracing.spec.js

```diff
@@ -52,5 +52,12 @@ module.exports.addTests = function({testRunner, expect}) {
       expect(error).toBeTruthy();
       await page.tracing.stop();
     });
+    it('should return a buffer', async({page, server, outputFile}) => {
+      await page.tracing.start({screenshots: true, path: outputFile});
```

Comment:
can we drop a necessity for the path as a part of this patch? This way we'll align with `page.screenshot` and `page.pdf`.

Reply:
@aslushnikov - sure, done.